code/game/atoms_movable.dm

```diff
@@ -727,11 +727,13 @@
 				break
 	. = dense_object_backup
 
-//called when a mob resists while inside a container that is itself inside something.
+//Called when something resists while this atom is its loc
+/atom/movable/proc/container_resist(mob/living/user)
+
```

Comment:
pop an empty return here and this should be good to go :)

Reply:
Done